docs/graphql/manual/api-reference/schema-metadata-api/remote-relationships.rst

```diff
@@ -0,0 +1,228 @@
+.. meta::
+   :description: Manage remote relationships with the Hasura schema/metadata API
+   :keywords: hasura, docs, schema/metadata API, API reference, remote joins, remote relationships
+
+Schema/Metadata API Reference: Remote Relationships
+===================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Remote Relationships will ensure independent remote schemas and related data types can be joined natively via Hasura.
+
+.. _create_remote_relationship:
+
+create_remote_relationship
+--------------------------
+
+``create_remote_relationship`` is used to create a new remote relationship with an existing remote schema.
+
+.. code-block:: http
+
+   POST /v1/query HTTP/1.1
+   Content-Type: application/json
+   X-Hasura-Role: admin
+
+  {
+    "type": "create_remote_relationship",
+    "args": {
+      "name": "sample_remote_relationship",
+      "table": "users",
+      "hasura_fields": [
+        "id"
+      ],
+      "remote_schema": "my-remote-schema",
+      "remote_field": {
+        "messages": {
```

Comment:
Will prefer a simpler `remote_field` definition here. Something which can be joined at top level.

Reply:
Something like:

```
{
  "message": {
    "arguments": {
      "message_id":"$id"
       }
    }
}
 
```